result/egressonlytracking.yaml

```diff
@@ -0,0 +1,250 @@
+components:
+  schemas:
+    EgressOnlyTracking.Metrics.Request:
+      description: >-
+        The container for a egress only tracking metric request.
+      type: object
+      properties:
+        port_names:
+          description: |-
+            Egress only tracking metrics will be retrieved for these port names.
```

Comment:
Maybe we should add . If no port_names are provided , egress_only_tracking stats will be returned for all ports which have one or more egress_only_tracking enabled .

Reply:
Added.